hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/resources/yarn-default.xml

```diff
@@ -5056,4 +5056,18 @@
     </description>
   </property>
 
+  <property>
+    <name>yarn.router.interceptor.allow-partial-result.enable</name>
+    <value>true</value>
```

Comment:
In the RBF PR there was controversy because the default behavior should be to now allow this.
I guess we can fix this in this PR.

Reply:
Thank you very much for your help reviewing the code!

Reply:
The old behavior was to not allow partial results, so this needs to be false.